src/openzaak/management/commands/send_test_notification.py

```diff
@@ -0,0 +1,39 @@
+from django.core.management.base import BaseCommand, CommandError
+
+from vng_api_common.notifications.kanalen import Kanaal
+from vng_api_common.notifications.models import NotificationsConfig
+
+
+class Command(BaseCommand):
+    help = "Send a test notification to verify if notifications are properly configured"
+
+    def handle(self, *args, **options):
+        nrc_client = NotificationsConfig.get_client()
+
+        kanaal_name = "test"
+        Kanaal(kanaal_name, "test")
+
+        data = {
+            "kanaal": kanaal_name,
```

Comment:
okay, but now it's actually going to fail unless the "test" kanaal has been registered :P

Something to add to the checklist in #440  - make sure that the `test` kanaal exists.